docs/articles/documentation/recipes/running-tests-with-saucelabs-on-travis-ci/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: docs
+title: Running Tests Using Travis CI and Sauce Labs
+permalink: /documentation/recipes/running-tests-using-travis-ci-and-sauce-labs/
+---
+# Running Tests Using Travis CI and Sauce Labs
+
+You can automatically run tests as a part of your build process using [Travis CI](https://travis-ci.org/).
+Furthermore, Travis CI integrates with [Sauce Labs](https://saucelabs.com/) that provides access to multiple real browsers in the cloud (see [Using Sauce Labs with Travis CI](https://docs.travis-ci.com/user/sauce-connect/)).
+Thus you can run your tests completely in the cloud.
```

Comment:
1.

> real browsers in the cloud

I have issues with this phrase... Though I understand what you mean.

2.

It's not quite good style to have two consecutive sentences end is the same way `in the cloud`.
We can easily go without it:

```
Furthermore, Travis CI integrates with [Sauce Labs](https://saucelabs.com/)
that provides access to multiple cloud browsers
(see [Using Sauce Labs with Travis CI](https://docs.travis-ci.com/user/sauce-connect/)).
Thus you can run your tests completely in the cloud.
```

Or if `cloud browsers` is not okay... we can come up with something else without sacrificing style.